release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -846,6 +846,16 @@ clouds:
 
 To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
 
+[id="ocp-4-12-hcp"]
+=== Hosted control planes (Technology Preview)
+
+[id="ocp-4-12-hcp-api"]
+==== HostedCluster and NodePool APIs are beta
+
+The default version for the `hypershift.openshift.io` API, which is the API for hosted control planes on {product-title}, is now v1beta1. If you work with the main branch, clear your `~/.kube/cache` directory so that `oc` and `kubectl` do not use outdated discovery data. Use a HyperShift CLI that is compiled from main so that your CLI works with the new version. The HyperShift installation gained 2 flags for webhooks. The default value of `true` for `--enable-conversion-webhook` makes it possible for v1alpha1 clients and previous {product-title} releases to work with v1beta1 resources on disk.
```

Comment:
>If you work with the main branch,
These are release notes for product so this statement does not apply here.

>The HyperShift installation gained 2 flags for webhooks. The default value of `true` for `--enable-conversion-webhook` makes it possible for v1alpha1 clients and previous {product-title} releases to work with v1beta1 resources on disk.

This statement can be removed. We provide an opinionated installation of the operator and that will have no webhooks enabled. Conversion Webhook was only enabled for internal consumers.
Instead we should mention here that transition for existing cluster from alpha to beta is not supported.

Reply:
Thanks so much for taking a look, @heliubj18, @jiezhao16, and @enxebre! I updated the release note according to your comments. If the text looks okay, let me know, and I'll move this forward in the publishing process.